cilium/rockcraft.yaml

```diff
@@ -41,6 +48,7 @@ parts:
     source-type: git
     source: https://github.com/cilium/proxy.git
     source-tag: v1.27
+    source-depth: 1
```

Comment:
These are all around the PR so we don't fetch the entire history of a git repo, saves us some disk space and clone time.